Sources/SourceKitLSP/LoggableMessageTypes.swift

```diff
@@ -12,66 +12,11 @@
 
 import Foundation
 import LSPLogging
-
-/// A request object, wrapping the parameters of a `RequestType` and tracking its state.
-public final class Request<R: RequestType> {
-
-  public typealias Params = R
-
-  public typealias Response = R.Response
-
-  /// The request id from the client.
-  public let id: RequestID
-
-  /// The client of the request.
-  public let clientID: ObjectIdentifier
-
-  /// The request parameters.
-  public let params: Params
-
-  private var replyBlock: (LSPResult<Response>) -> Void
-
-  /// Whether a reply has been made. Every request must reply exactly once.
-  private var replied: Bool = false {
-    willSet {
-      precondition(replied == false, "replied to \(id) more than once")
-    }
-  }
-
-  public init(
-    _ request: Params,
-    id: RequestID,
-    clientID: ObjectIdentifier,
-    reply: @escaping (LSPResult<Response>) -> Void
-  ) {
-    self.id = id
-    self.clientID = clientID
-    self.params = request
-    self.replyBlock = reply
-  }
-
-  deinit {
-    precondition(replied, "request \(id) never received a reply")
-  }
-
-  /// Reply to the request with `result`.
-  ///
-  /// This must be called exactly once for each request.
-  public func reply(_ result: LSPResult<Response>) {
-    replied = true
-    replyBlock(result)
-  }
-
-  /// Reply to the request with `.success(result)`.
-  public func reply(_ result: Response) {
-    reply(.success(result))
-  }
-}
+import LanguageServerProtocol
 
 fileprivate extension Encodable {
   var prettyPrintJSON: String {
     let encoder = JSONEncoder()
-    encoder.outputFormatting.insert(.sortedKeys)
```

Comment:
Did you mean to remove this?

Reply:
I did not, must have been caused by rebasing. Thanks for catching.